src/components/home/Configuration.tsx

```diff
@@ -45,191 +45,595 @@ const CodeTabItem = ({code, language}: {code: string; language: "json" | "yaml"
 
 export default Configuration
 
-const GRAPHQL_CONFIG = `schema
-  @server(port: 8000)
-  @upstream(baseURL: "http://jsonplaceholder.typicode.com") {
+const GRAPHQL_CONFIG = `schema 
+@server @upstream(baseURL: "https://jsonplaceholder.typicode.com") {
   query: Query
 }
 
-type Query {
-  users: [User] @http(path: "/users")
-  posts: [Post] @http(path: "/posts")
+type Address {
+  city: String
+  geo: Geo
+  street: String
+  suite: String
+  zipcode: String
 }
 
-type User {
-  id: Int!
-  name: String!
-  username: String!
-  email: String!
+type Company {
+  bs: String
+  catchPhrase: String
+  name: String
 }
 
+type Geo {
+  lat: String
+  lng: String
+}
+
+type Photo {
+  albumId: Int
+  id: Int
+  thumbnailUrl: String
+  title: String
+  url: String
+  album: Album @call(steps: [{query: "album", args: {id: "{{.value.albumId}}"}}])
+}
 
 type Post {
-  id: Int!
-  title: String!
-  body: String!
-  userId: Int!
+  body: String
+  id: Int
+  title: String
+  userId: Int
+  user: User @call(steps: [{query: "user", args: {id: "{{.value.userId}}"}}])
+  comments: [Comment] @call(steps: [{query: "comments"}])
+}
 
-  # Expand a post with user information
-  user: User @http(path: "/users/{{.value.userId}}")
+type Comment {
+  body: String
+  email: String
+  id: Int
+  name: String
+  postId: Int
+  post: Post @call(steps: [{query: "post", args: {id: "{{.value.postId}}"}}])
+}
+
+type Query {
+  album(id: Int!): Album @http(path: "/albums/{{.args.id}}")
+  albums: [Album] @http(path: "/albums")
+  comment(id: Int!): Comment @http(path: "/comments/{{.args.id}}")
+  comments: [Comment] @http(path: "/comments")
+  photo(id: Int!): Photo @http(path: "/photos/{{.args.id}}")
+  photos: [Photo] @http(path: "/photos")
+  post(id: Int!): Post @http(path: "/posts/{{.args.id}}")
+  posts: [Post] @http(path: "/posts")
+  todo(id: Int!): Todo @http(path: "/todos/{{.args.id}}")
+  todos: [Todo] @http(path: "/todos")
+  user(id: Int!): User @http(path: "/users/{{.args.id}}")
+  users: [User] @http(path: "/users")
+}
+
+type Todo {
+  completed: Boolean
+  id: Int
+  title: String
+  userId: Int
+  user: User @call(steps: [{query: "user", args: {id: "{{.value.userId}}"}}])
+}
+
+type User {
+  address: Address
+  company: Company
+  email: String
+  id: Int
+  name: String
+  phone: String
+  username: String
+  website: String
+  posts: [Post] @call(steps: [{query: "posts"}])
+  todos: [Todo] @call(steps: [{query: "todos"}])
```

Comment:
All the `call` operators are incorrect. They just return all the posts and all the todos.